### PR TITLE
Update World Romance Translation #4007

### PR DESCRIPTION
**Pull request by ghost**
Update Metadata and Add Genre Page to presence

### CODE REVIEW

File: websites/W/World Romance Translation/dist/metadata.json [hunk 1]
```diff
@@ -5,14 +5,21 @@
     "name": "वाह्य",
     "id": "819696394180100106"
   },
+  "contributors": [
+    {
+      "name": "Kirito",
+      "id": "859291951614525440"
+    }
+  ],
```
Comment (ririxidev) [inline websites/W/World Romance Translation/dist/metadata.json:13]:
```suggestion
```
not enough changes

File: websites/W/World Romance Translation/dist/metadata.json [hunk 2]
```diff
@@ -5,14 +5,21 @@
...
   "category": "other",
   "logo": "https://i.imgur.com/WJFdTQL.png",
   "thumbnail": "https://i.imgur.com/s3wgvCP.png",
-  "version": "1.0.0",
-  "color": "#db75b1",
-  "tags": ["manga", "indonesian", "romance"],
+  "version": "1.1.0",
```
Comment (ririxidev) [inline websites/W/World Romance Translation/dist/metadata.json:17]:
```suggestion
  "version": "1.0.1",
```

File: websites/W/World Romance Translation/presence.ts [hunk 3]
```diff
@@ -17,11 +17,18 @@ presence.on("UpdateData", async () => {
     if (searchQuery === null) presenceData.details = "Browsing Home Page";
```
Comment (ririxidev) [inline websites/W/World Romance Translation/presence.ts:17]: then change this english text there too?

**Review by rxri — CHANGES_REQUESTED** [review]

**Review by ghost — COMMENTED** [review]
Translate Browsing HomePage to indonesia

File: websites/W/World Romance Translation/presence.ts (hunk 3 above)
Comment (irfan1810) [reply websites/W/World Romance Translation/presence.ts]: i have changed it

File: websites/W/World Romance Translation/presence.ts [hunk 4]
```diff
@@ -31,42 +38,43 @@ presence.on("UpdateData", async () => {
...
   else if (pathname === "/contact-us/")
-    presenceData.details = "Viewing Contact Us Page";
+    presenceData.details = "Membuka Halaman Contact Us";
   else if (pathname === "/privacy-policy/")
-    presenceData.details = "Viewing Privacy Policy";
-  else if (pathname === "/dmca/") presenceData.details = "Viewing DMCA Policy";
+    presenceData.details = "Melihat Kebijakan Privasi";
+  else if (pathname === "/dmca/") presenceData.details = "Melihat DMCA";
 
   if (presenceData.details === null) {
```
Comment (SlowLife1661) [inline websites/W/World Romance Translation/presence.ts:79]:
```suggestion
  if (!presenceData.details) {
```

File: websites/W/World Romance Translation/presence.ts [hunk 5]
```diff
@@ -31,44 +38,45 @@ presence.on("UpdateData", async () => {
         "select#select-paged"
       );
     if (mangaName !== null && chapterNumber !== null && pageNumber !== null) {
-      presenceData.details = `Reading ${mangaName.innerText.substring(
+      presenceData.details = `Membaca ${mangaName.innerText.substring(
         0,
         mangaName.innerText.indexOf("Chapter")
       )}`;
-      presenceData.state = `${chapterNumber.selectedOptions[0].innerText} Page ${pageNumber.selectedOptions[0].innerText}`;
+        presenceData.state = `${chapterNumber.selectedOptions[0].innerText} Slide ${pageNumber.selectedOptions[0].innerText}`;
```
Comment (SlowLife1661) [inline websites/W/World Romance Translation/presence.ts:45]:
```suggestion
      presenceData.state = `${chapterNumber.selectedOptions[0].innerText} Slide ${pageNumber.selectedOptions[0].innerText}`;
```

**Review by Slowlife01 — DISMISSED** [review]

**Review by rxri — APPROVED** [review]

**Review by Slowlife01 — APPROVED** [review]

**Pull request merged**